crates/lsp/src/main_loop.rs

```diff
@@ -232,7 +237,7 @@ impl GlobalState {
         // tower-lsp backend and the Jupyter kernel.
         let (events_tx, events_rx) = tokio_unbounded_channel::<Event>();
 
-        let (log_state, log_tx) = LogState::new(client.clone());
+        let (log_thread_state, log_tx) = LogThreadState::new(client.clone());
```

Comment:
Just a rename so we can use `LogState` for the bit that holds the reload handle

This thread state thing will also eventually go away in lsp-server world